src/edu/stanford/nlp/sempre/InteractiveMaster.java

```diff
@@ -0,0 +1,328 @@
+package edu.stanford.nlp.sempre;
```

Comment:
I moved BeamFloating Parser to interactive as well, and added a bunch of getter and setters, or changing to protected/public as needed.